tsconfig.json

```diff
@@ -25,6 +25,7 @@
     "resolveJsonModule": true,
     "skipLibCheck": true,
     "strict": true,
+    "strictPropertyInitialization": false,
```

Comment:
This is so TS variables can be declared without default values, such as `proxyUrl: string` and `appName: string`

Reply:
is this actually necessary? 
https://github.com/DataBiosphere/terra-ui/blob/400d461d59e36772110343560b0146f05c39be22/src/libs/ajax/data-table-providers/WdsDataTableProvider.ts#L92
doesn't seem to need it